policy/modules/contrib/wireguard.te

```diff
@@ -0,0 +1,49 @@
+policy_module(wireguard, 1.0.0)
+
+########################################
+#
+# Declarations
+#
+
+type wireguard_t;
+type wireguard_exec_t;
+init_daemon_domain(wireguard_t, wireguard_exec_t)
+
+type wireguard_unit_file_t;
+systemd_unit_file(wireguard_unit_file_t)
+
+########################################
+#
+# wireguard local policy
+#
+allow wireguard_t self:capability { net_admin };
+allow wireguard_t self:fifo_file rw_fifo_file_perms;
+allow wireguard_t self:netlink_generic_socket create_socket_perms;
+allow wireguard_t self:netlink_netfilter_socket create_socket_perms;
+allow wireguard_t self:netlink_route_socket rw_netlink_socket_perms;
```

Comment:
There seem to be inconsistent permissions sets in obj_perm_sets.spt, if create is required, I'd rather use `create_netlink_socket_perms`; otherwise it is good.

Reply:
getting this error for netlink_generic_socket and netlink_netfilter_socket:

permission nlmsg_write is not defined for class netlink_netfilter_socket' 

So I let there create_socket_perms

Reply:
That's correct, but netlink_route_socket should have create_netlink_socket_perms.